src/containers/MessagingContainer.js

```diff
@@ -2,12 +2,56 @@ import React from 'react'
 import {Redirect} from 'react-router-dom'
 import {connect} from 'react-redux'
 import {fetchUserChatrooms} from '../actions/useractions'
+import ChatroomPreview from '../components/Messages/ChatroomPreview'
+import Chatroom from '../components/Messages/Chatroom'
+import {Grid} from 'semantic-ui-react'
+import NewMessage from '../components/Messages/NewMessage'
 class MessagingContainer extends React.Component {
+  
+  state = {
+    selectedChatroom: null
+  }
+  
+
+  handleChatroomShow = (chatroomId) => {
+    const chatroom = this.props.chatrooms.find(cr => cr.id === chatroomId)
+    this.setState({selectedChatroom: chatroom})
+  }
+
+  orderChatrooms = (chatrooms) => {
+    if (chatrooms.length > 0){
+      return chatrooms.sort((cr, cr2) => (cr.updated_at < cr2.updated_at) ? 1 : -1)
+    }else{
+      return []
+    }
+  }
   render(){
-    console.log(this.props)
     if (sessionStorage.jwt){
       return(
-        <h1>Messages</h1>
+        <>
+          <Grid className="messages-container" centered style={{maxWidth: "1000px", margin: "auto"}}>
```

Comment:
add to styles folder

Reply:
also, you should check out [css grid](https://css-tricks.com/snippets/css/complete-guide-grid/). that, combined with [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) pretty much eliminates our need for semantic UI, other than their icon library